proto/one/operations/model/model_template.proto

```diff
@@ -27,18 +22,12 @@ message ModelTemplate {
     string outputDataSourceDriver = 9;
     // Id of the dataSet containing the output values
     google.protobuf.StringValue outputDataSetId = 10;
-    // Locations that house the parameters
-    repeated ModelLocation locations = 11;
-    // Current input parameters of the model
-    repeated TemplateParameter inputs = 12;
-    // Current output parameters of the model
-    repeated TemplateParameter outputs = 13;
-    // Comment
-    google.protobuf.StringValue comment = 14;
     // Specifies if model is enabled (defaults to false)
     bool enabled = 15;
-	// Setpoints
-    repeated TemplateParameter optimizationInputs = 16;
+
+    oneof modelDetails {
+        DelphiModelDetail modelDetail = 16;
```

Comment:
I think this field name should be `delphiModelDetails`

Reply:
Sorry @josephquiroz the oneof should be modelDetails as it is not specific to Delphi and we plan to add more fields to the oneof as other implementations are created.  The DelphiModelDetail field that is a field in the oneof should be named delphiModelDetails